addon/helpers/github-issue.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export function githubIssue([title]/*, hash*/) {
+	let titleStr = encodeURIComponent(`Demo App Test fail: ${title}`);
+	let linkStr = 
+		`https://github.com/nathanhammond/ember-a11y/issues/new?title=${titleStr}`;
+  return linkStr;
```

Comment:
Did you know that you can return a DOM node from a helper? `document.createElement('a')` works here. :) You can use that to move more of the markup into the helper.